dll/keyboard/kbdsf/kbdsf.c

```diff
@@ -0,0 +1,479 @@
+/*
```

Comment:
Can you tell me whether this file has been encoded in UTF-8 or not? (If not, please do that.)

Reply:
The file is already UTF8 encoded

Reply:
ok thanks. (I was asking because I saw characters that were strangely displayed here.)